src/app/components/SubjectHeading/BibsList.jsx

```diff
@@ -145,13 +150,15 @@ class BibsList extends React.Component {
       );
     }
 
+    const h2Text = `Viewing ${this.firstBib() + 1} - ${this.lastBib()} of ${total || ''} items for Heading "${label}"`;
+
     return (
       <div
         className="nypl-column-half bibsList"
         tabIndex='0'
         aria-label="Titles related to this Subject Heading"
       >
-        <h2 id="titles">Titles</h2>
+        <h2 id="titles">{h2Text}</h2>
```

Comment:
Not sure if this is still quite a header. But not a big deal. The font looks way too big. Also, it's a little hard to tell if it's just wrapping or the design, but on the Figma, the subject heading is on a separate line.

Reply:
Changed the font size to 1.2rem following discussion with Ellen

Reply:
👍